projects/angular/src/layout/nav/_variables.subnav.scss

```
@@ -6,14 +6,15 @@
  */
 @use '../../utils/variables/variables';
 @use '@cds/core/tokens/tokens.scss';
+@use '../../layout/density/' as density;
 
 // Usage: ../layout/nav/_subnav.clarity.scss
 // TODO: would be nice to key text color in subnav off of the bgColor...
 $clr-subnav-bgColor: var(--clr-subnav-bg-color) !default;
 
 // Usage: ../layout/nav/_subnav.clarity.scss
 // Usage: ../layout/nav/_nav.clarity.scss
-$clr-subnav-height: tokens.$cds-global-space-11 !default;
+$clr-subnav-height: density.$clr-base-layout-space-xl !default;
```

Talk to designers to change the heigth in figma with more sutable `row-heigth-m`

```suggestion
$clr-subnav-height: density.$clr-base-row-heigth-m !default;
```